.github/workflows/set_wheel_build_matrix.yml

```diff
@@ -52,7 +52,7 @@ jobs:
 
       - name: Kokkos version
         id: kokkos_version
-        run: echo "kokkos_version=[\"4.2.00\"]" >> $GITHUB_OUTPUT
+        run: echo "kokkos_version=[\"4.3.01\"]" >> $GITHUB_OUTPUT
```

Comment:
Could you use the `ci::build_wheels` GH label to check wheels with this update?

Reply:
Will do.